src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -59,6 +59,7 @@ export type UpdateQueue = {
   first: Update | null,
   last: Update | null,
   hasForceUpdate: boolean,
+  callbackList: null | Array<Callback>,
```

Comment:
This used to use a linked list like all the other structures. That requires dynamic reallocation as it grows. It also requires us to take on a dependency on such an implementation. This might matter if we make a WASM port.

Would it make sense to just use a simple linked list here too, just for consistency?